app/background/wallet/service.js

```diff
@@ -446,6 +447,12 @@ class WalletService {
     await finalizeMany(wallet, names);
   };
 
+  renewMany = async (names) => {
```

Comment:
not blocking for this PR - now that we have ledger, we should add `ledgerProxy` to all calls like this. We will need to separate `renewMany` to `createRenewMany` and `sendRenewMany` for `ledgerProxy`.